plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -83,6 +86,14 @@ public class AndroidCommandPluginExtension {
         monkey
     }
 
+    void script(Action<NamedDomainObjectContainer<InputSpec>> action) {
+        action.execute(scripts)
```

Comment:
I guess so. But I kinda need to execute the action to retrieve the name to create the task. 

In the following example, I need to get the name `autoLogin` in advance to create the task with that name but still need to run the rest lazily. How would I do that? 

By doing `action.execute`, the script container gets an `InputSpec` object with a name. 

```
autoLogin {
        2.times {
            text 'bob'
            enter()
        }
        enter()
    }
```